webpack.common.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+const WebpackNotifierPlugin = require("webpack-notifier");
+const webpack = require("webpack");
+const path = require("path");
+const absolute = (relPath) => path.join(__dirname, relPath);
+const EnvLoaderPlugin = require("webpack-env-loader-plugin");
+
+let env = process.env.NODE_ENV || "development";
```

Comment:
I think we want the requires to stay in-sync with what's in the file.  It's not clear to me that having it in sync with the other file makes any difference.